### PR TITLE
fix parameter loading bug

### PR DESCRIPTION
**Pull request by mutdmour**

### CODE REVIEW

File: packages/editor-ui/src/components/ParameterInput.vue [hunk 1]
```diff
@@ -549,7 +549,6 @@ export default mixins(
 					}
 
 					for (const checkValue of checkValues) {
-						if (checkValue !== undefined && checkValue.includes(CUSTOM_API_CALL_KEY)) continue;
```
Comment (mutdmour) [inline packages/editor-ui/src/components/ParameterInput.vue:552]: duplicate check. already handled by https://github.com/n8n-io/n8n/pull/3374/files#diff-f360567622b0c182cf21c3df2bc9947d2c4a8db42ff225b353431d79be1360ddL543

**Review by ivov — APPROVED** [review]
Load options working, Custom API Call option not flagged.

**Comment by janober** [comment]
Thanks a lot. Got merged.

**Comment by janober** [comment]
Got released with `n8n@0.178.2`

**Pull request merged**